### DISCUSSION TITLE
What means “increments” option value in Strapi model configuration?

### DISCUSSION THREAD

**Discussion by NicoMachado** [strapi/strapi#7354] [Legacy Topics]
I'm quite new to Strapi, but I found it very usefull!

I'm getting data from existing database, so I don't need/want to change anything in the tables. I'm using "false" to timestamp option, but there is another option parameter called "increments", there is no reference to that in the docs.

Anyone have any idea what is this parameter for ?

```json
{
  "kind": "collectionType",
  "collectionName": "filiales",
  "info": {
    "name": "Filial"
  },
  "options": {
    "increments": true,
    "timestamps": false
  },
  "attributes": { 
   ...
   }
```
Best Regards

**Comment by derrickmehaffy** [comment]
Hello @NicoMachado in this case the `increments` are the default `id` field

**Reply by NicoMachado** [reply]
Hi, @derrickmehaffy , and what is the "false" meaning to that ?
Best regards

**Reply by derrickmehaffy** [reply]
In this case it's likely technical debt, I tried to set it as false and removed the key and it didn't change anything. Reviewing the codebase shows it may no longer be used but I will ping @alexandrebodin and @petersg83 to get their thoughts.

It's possible we could just remove this from the codebase.

**Reply by NicoMachado** [reply]
Excelent!
Thanks!
